app.territoiresentransitions.react/src/app/pages/collectivite/PlansActions/FicheAction/etapes/etape/etape.tsx

```diff
@@ -0,0 +1,135 @@
+import { useState } from 'react';
+import classNames from 'classnames';
+import { useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+
+import { RouterOutput } from '@/api/utils/trpc/client';
+
+import { Button, Checkbox } from '@/ui';
+
+import { useEtapesDispatch } from '../etapes-context';
+import ModalDeleteEtape from './modal-delete-etape';
+import { Textarea } from './textarea';
+import { useUpsertEtape } from './use-upsert-etape';
+
+type Props = {
+  etape: RouterOutput['plans']['fiches']['etapes']['list'][0];
+  isReadonly: boolean;
+};
+
+export const Etape = ({ etape, isReadonly }: Props) => {
+  const {
+    attributes,
+    listeners,
+    setNodeRef,
+    transform,
+    transition,
+    isDragging,
+  } = useSortable({
+    id: etape.id,
+    disabled: isReadonly,
+  });
+
+  // permet de corriger un bug de hauteur avec le scale de l'élêment qui est drag
+  const custromTransform = transform
+    ? isDragging
+      ? { ...transform, scaleY: 1 } // renforce le scale
+      : transform
+    : null;
+
+  const style = {
+    transform: CSS.Transform.toString(custromTransform),
+    transition,
+    opacity: isDragging ? 0.5 : 1,
+  };
+
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+
+  const dispatchEtapes = useEtapesDispatch();
+  const { mutate: updateEtape } = useUpsertEtape();
+
+  return (
+    <div
+      ref={setNodeRef}
+      style={style}
+      {...attributes}
+      {...listeners}
+      className={classNames(
+        'group relative flex items-start w-full p-4 rounded-lg',
+
+        {
+          'bg-error-2 bg-opacity-50': isDeleteModalOpen,
+          'hover:bg-grey-2': !isReadonly,
+          'cursor-default': isReadonly,
+          'z-10 hover:cursor-grabbing': isDragging,
```

Comment:
Je pense que niveau découvrabilité du drag and drop, il vaudrait mieux toujours avoir ce curseur activé sur la carte, ou alors utiliser les drag handle de la librairie

Reply:
Oui je suis d'accord avec le handle sur le côté mais pas sur les design et je n'ai pas le temps de faire un truc bien direct en code, je préfère ship ça déjà. Ça reviendra surement dans les améliorations.

Pour le curseur je ne suis pas pour, car dans l'étape il faut avoir celui du texte pour modifier l'étape et celui de la checkbox